### DISCUSSION TITLE
Taskbar transparent

### DISCUSSION THREAD

**Discussion by novatiberium** [valinet/ExplorerPatcher#953] [Q&A]
While using ExplorerPatcher on my Desktop PC, the Windows 10 taskbar is transparent (and also the OS freezes apps when it is in use, no notifications shown, no flyouts, etc) - while using it on my laptop have no issues running with the same Windows 11.

Is there a potential fix?

**Comment by valinet** [comment]
What Windows build? At least mention that…

**Reply by novatiberium** [reply]
Windows 11 build 22563.1. Both of my computers run this build, the laptop runs EP fine but desktop is another story...

**Reply by valinet** [reply]
EP is not tested on Insider Preview builds.

**Comment by memeisms** [comment]
I was able to resolve this issue by clicking the "Restore Default Settings" button on the "About" tab of ExplorerPatcher, this forced the Win10 taskbar to reappear, as well as solved the explorer crashing error when trying to open the notifications panel. YMMV of course, but this is what worked for me.

**Comment by hawkeyexp** [comment]
Build 22572 has same problem but reset defaults won't work. Taskbar is fully transparent :-)

**Reply by valinet** [reply]
That doesn’t mean reset defaults doesn’t work.

**Reply by apazzy** [reply]
Same build, transparent Win10 taskbar on Win11. Resetting to defaults did not resolve the issue. It _is_ an insider preview, and you said above it wasn't tested on insider builds.

Edition	Windows 11 Pro
Version	22H2
OS build	22572.1
Experience	Windows Feature Experience Pack 1000.22572.1.0

**Reply by memeisms** [reply]
Sorry, I should've mentioned I'm on dev build 22567.ni_release.220225-1325, so theoretically none of this should be working for me, but in my specific case, resetting defaults solved my issue.

**Comment by sirthommo** [comment]
I'm also suddenly having this problem. Restoring default settings didn't fix the transparency issue. The issue only occurred for me with the last update. 

EP Version 22621.608.51.3

Edition	Windows 11 Home
Version	22H2
Installed on	‎01/‎11/‎2022
OS build	22621.1413
Experience	Windows Feature Experience Pack 1000.22639.1000.0

**Reply by valinet** [reply]
What's the point of reporting bugs when using an outdated EP version?

**Reply by sirthommo** [reply]
To be honest, I didn't even know it was outdated because I even clicked on Check for updates and it came back with nothing.

**Reply by sirthommo** [reply]
![image](https://user-images.githubusercontent.com/128139867/225973843-edfbfdd8-cfd4-47fd-a30f-53c4b104ae1f.png)

Updated to that version manually.

![image](https://user-images.githubusercontent.com/128139867/225973982-d36f37cd-81bd-406d-858d-cae1c41e705b.png)

Task Bar still transparent.

**Reply by pyrates999** [reply]
Check your notifications after you click on "check for updates".

Then click on "Update program and restart file explorer".

**Reply by pyrates999** [reply]
Did you install this?

https://apps.microsoft.com/store/detail/translucenttb/9PF4KZ2VN4W9

**Comment by michaelmuselik** [comment]
How can I set the transparent taskbar :-) ?

**Reply by plofhaan** [reply]
https://github.com/TranslucentTB/TranslucentTB

**Reply by pyrates999** [reply]
I like nice taskbar: https://www.microsoft.com/store/productId/9PKL2S93XWB5

You can adjust the transparency.  I prefer 50% as there is some transparency, but I can still see the icons on my taskbar.

**Comment by skykharkov** [comment]
Bug confirmed.
Restarting, updating, reset settings - not help, unfortunately. 

![Screenshot_1](https://github.com/valinet/ExplorerPatcher/assets/20350627/f2c14160-84ad-48a9-98ce-04cee5f091b5)

Here is how it looked for me.

![Screenshot_2](https://github.com/valinet/ExplorerPatcher/assets/20350627/73c853af-3211-4fb5-b349-d4f53f98ae1f)

**Reply by claudioschell** [reply]
Same. Maybe it was after a Windows upgrade:
Mise à jour cumulative pour Windows 11 Version 23H2 pour les systèmes x64 (KB5035853)

**Comment by paskausks** [comment]
Hello!

Same issue here. Settings reset didn't help.

![image](https://github.com/valinet/ExplorerPatcher/assets/5221695/cdbcefab-6721-41b0-b798-4b4d506ff7d1)

Edition	Windows 11 Home
Version	23H2
Installed on	‎26.‎02.‎2023
OS build	22631.3296
Experience	Windows Feature Experience Pack 1000.22687.1000.0

**Comment by jwgn** [comment]
Same here

Microsoft Windows [Version 10.0.22631.3296]

Fixed in Pre-Release 22621.3296.64.1 https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3296.64.1_9e9c016

**Comment by Apollyon69** [comment]
Reporting, the issue happens also here, it was working as expected, Windows rebooted after 2024-03 Cumulative Update for Windows 11 Version 22H2 for x64-based Systems (KB5035853), updated ExplorerPatcher but taskbar turned transparent and stays that way. The only workaround is to use the horrible Windows 11 taskbar style, but then I cannot place the taskbar at top, as I prefer.

Update, the prerelease here  ^^^ fixed the issue, thanks

**Reply by descentfan1975** [reply]
Agreed.  Can confirm pre-release version fixed the issue.

**Reply by DiePutinDie** [reply]
Same exact issue after the 03-2024 Cumulative update.

Can also confirm the prerelease [available here](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3296.64.1_9e9c016) fixed the issue :))

**Reply by distrion-d** [reply]
Worked for me as well

**Reply by jdpm1991** [reply]
Worked for me as well

**Comment by skalsky0** [comment]
Seeing this same issue after upgrading to Version 22621.2861.62.2.  Restoring default settings didn't resolve the problem.

**Reply by skalsky0** [reply]
Pre-release also fixed me.   Version 22621.3296.64.1

**Comment by jdpm1991** [comment]
This just happened to me I can see my wallpaper through my taskbar

**Comment by yozi66** [comment]
Same for me. Workaround: painted a gray strip at the bottom of the wallpaper.

**Reply by yozi66** [reply]
Version 22621.3296.64.1 fixed the problem. Thanks

**Comment by lj119621** [comment]
same for me by yesterday's windows update. 
and Fixed well in Pre-Release 22621.3296.64.1
https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3296.64.1_9e9c016

**Reply by paskausks** [reply]
yeah, updating this morning fixed it for me.

thanks to the devs!

**Comment by d-0-s-t** [comment]
The latest update fixed this, but didn't want it to :). Posts above suggest few tools that can make taskbar transparent. Can it be done with  Explorer Patcher?

**Comment by pyrates999** [comment]
This has already been declined by the developers of EP: https://github.com/valinet/ExplorerPatcher/issues/2993

Install one of the suggest applications instead that already does this.
